sheet-to-json.js

```diff
@@ -50,12 +53,37 @@ function sortObjByKey(value) {
     value;
 }
 
+function getDelta(state) {
+  return  rawData.raw_data.reduce((stat, row) => {
+    let stateName = row.detectedstate;
+    let isToday = moment().isSame(moment(row.dateannounced, "DD-MM-YYYY"), "day");
+    if (stateName && (stateName === state || state === "Total") && isToday) {
+      let currentStatus = row.currentstatus;
+      if (currentStatus) {
```

Comment:
put a console.error for if currentStatus is empty or not one of the three in the sheet, 
We will know that sheet needs to be corrected basis this log.

Reply:
@sumitmckv added empty check for currentStatus